app/src/main/assets/queries/nearby_query.rq

```diff
@@ -13,7 +13,7 @@ SELECT
      SERVICE wikibase:around {
        ?item wdt:P625 ?location.
        bd:serviceParam wikibase:center "Point(${LONG} ${LAT})"^^geo:wktLiteral.
-       bd:serviceParam wikibase:radius "${RADIUS}" . # Radius in kilometers.
+       bd:serviceParam wikibase:radius "${RAD}" . # Radius in kilometers.
```

Comment:
In your [previous PR](https://github.com/commons-app/apps-android-commons/commit/04ea68333b35982e682b03564a37ba3b3c49e9c6) I guess you forgot to make the change in the `nearby_query.rb`, so the nearby places stopped loading. Can you test it thoroughly once? I will be happy to merge it then. :)